packages/zend-xml/library/Zend/Xml/Security.php

```diff
@@ -19,7 +19,7 @@
  * @version    $Id$
  */
 
- 
+
```

Comment:
unnecessary blank line added

Reply:
Actually the line was already there, an unnecessary white space was removed.

Reply:
oh my bad 🙈 anyway, let's avoid unnecessary changes in code formatting